### DISCUSSION TITLE
Can I host a meshroom workflow online, in heroku?

### DISCUSSION THREAD

**Discussion by Muhammad-1990** [alicevision/Meshroom#1474] [General]
Is it possible to host in heroku and trigger a workflow?
I don't have much computational power, and I would just like some server to do the work for me and give me a downloadable output.

basically do a batch import of images and have meshroom process it.
use images to reconstruct a 3d point cloud and extract a 3d mesh or save the project and make it downloadable, after the heavy duty processing  has been done.

thanks,

**Comment by natowi** [comment]
I suppose you could run Meshroom on heroku, but without the DepthMap node, which requires a NVIDIA GPU heroku does not provide. Alternatives are: https://github.com/alicevision/meshroom/wiki/Meshroom-in-Google-Colab-(cloud) and https://github.com/alicevision/meshroom/wiki/Meshroom-on-AWS

You could also test https://vgputestdrive.nvidia.com/accounts/register/ (NVIDIA Cloud VM with GPU), but I don´t know if it is still available.

**Reply by Muhammad-1990** [reply]
Thank you! Exactly what i was looking for. Im a newbie here and guess i was searching using the wrong keywords. Anyways drop a link if i can buy you a coffee. Very much appreciated!

**Reply by natowi** [reply]
Happy to help. I´m always interested to see what Meshroom is being used for, so feel free to share if you want to. 
If you use and love AliceVision and Meshroom you can support us here if you like: https://alicevision.org/association/#donate
